content/en/docs/instrumentation/net/automatic/_index.md

```diff
@@ -0,0 +1,275 @@
+elements inside the `<aspNetCore>` block of your `Web.config` file to set
+configuration via environment variables.
+
+### Advanced configuration
+
+You can add the
+[`<environmentVariables>`](https://docs.microsoft.com/en-us/iis/configuration/system.applicationhost/applicationpools/add/environmentvariables/)
+in `applicationHost.config` to set environment variables for given application
+pools.
+
+Consider setting common environment variables, for all applications deployed to
+IIS by setting the environment variables for `W3SVC` and `WAS` Windows Services.
+
+{{% alert title="Note" color="note" %}} For IIS versions older than 10.0, you
+can consider creating a distinct user, set its environment variables and use it
+as the application pool user. {{% /alert %}}
+
+## NuGet package
+
+You can instrument
+[`self-contained`](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained)
+applications using the NuGet packages. See [NuGet packages](./nuget-packages)
+for more information.
+
+## Instrument a container
+
+For an example of Docker container instrumentation, see
+[the example](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/tree/main/examples/demo)
+on GitHub.
+
+You can also use the
+[Kubernetes Operator for OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-operator).
+
+## Configuring the agent
+
+To see the full range of configuration options, see
+[Configuration and settings](./config).
+
+## Log to trace correlation
+
+{{% alert title="Note" color="note" %}} Automatic log to trace correlation
+provided by OpenTelemetry .NET Automatic Instrumentation currently works only
+for .NET applications using `Microsoft.Extensions.Logging`. See
+[#2310](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2310)
+for more details. {{% /alert %}}
+
+OpenTelemetry .NET SDK automatically correlates logs to trace data. When logs
+are emitted in the context of an active trace, trace context
+[fields](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#trace-context-fields)
+`TraceId`, `SpanId`, `TraceState` are automatically populated.
+
+The following are logs produced by the sample console application:
+
+```json
+"logRecords": [
+    {
+        "timeUnixNano": "1679392614538226700",
+        "severityNumber": 9,
+        "severityText": "Information",
+        "body": {
+            "stringValue": "Success! Today is: {Date:MMMM dd, yyyy}"
+        },
+        "flags": 1,
+        "traceId": "21df288eada1ce4ace6c40f39a6d7ce1",
+        "spanId": "a80119e5a05fed5a"
+    }
+]
+```
+
+For more information, see:
+
+- [OpenTelemetry .NET SDK](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/correlation)
+- [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#trace-context-fields)
```

Comment:
```suggestion
- [OpenTelemetry Specification](/docs/specs/otel/logs/data-model/#trace-context-fields)
```

give this a try.

Reply:
Trying now with some Regex magic...

Reply:
The issue persists. I've noticed there are like 1k such links in the repo. Are those checks we can just ignore?